.github/workflows/pythonpublish.yml

```diff
@@ -0,0 +1,31 @@
+# This workflows will upload a Python Package using Twine when a release is created
+# For more information see: https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#publishing-to-package-registries
+
+name: Upload Python Package
+
+on:
+  release:
+    types: [created]
```

Comment:
I think we should use `[published]` here: https://github.community/t5/GitHub-Actions/What-is-the-created-type-in-on-release/td-p/41171

Reply:
yep, you are right.